modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -206,3 +206,13 @@ extension Set: RequestDecodable where Element: Content {
 extension Set: Content where Element: Content { }
 
 extension AnyCodable: Content {}{{/useVapor}}
+
+extension JSONEncodable where Self: Encodable {
+    func encodeToJSON() -> Any {
+        let encoder = JSONEncoder()
```

Comment:
@0x0c here instead of creating a new JSONEncoder, please use `CodableHelper.jsonEncoder`

Reply:
Done with https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/1c06b4d09e94698eeba688c5fa3e1d30ea8b18ec